dahu/core/app/scripts/templates/layouts/presentation/screencast.html

```diff
@@ -49,31 +49,34 @@
     <body>
     <div class="deck-container">
         <div id="myPresentation">
-            <!-- iterate on all the screens -->
-            <section class="slide">
-            </section>
-            {{#each this.screencast.attributes.screens.models}}
-            <section class="slide" id="screen_{{id}}">
-                <!-- iterate on all objects, show only images -->
-                {{#each attributes.objects.models}}
-                {{#if isImage }}
-                <img src="{{attributes.img}}" class="background" style="width:{{screencastWidth}}; height:{{screencastHeight}};">
-                {{/if}}
-                {{#if isTooltip }}
-                <div class="slide">
-                    <div class="tooltip"
-                         style="background-color: {{attributes.color}}; width: {{attributes.width}}; top:{{normalizedToPixel (screencastHeight) attributes.posy}}; left: {{normalizedToPixel (screencastWidth) attributes.posx}};">
-                        {{{attributes.text}}}
+            {{#if this.screencast.attributes.screens.models}}
```

Comment:
Should you not test .length() > 0 ? What happen if the collection is initialised but contains 0 elements ?